rero_ils/modules/patron_transactions/api.py

```diff
@@ -238,6 +231,13 @@ class PatronTransactionsIndexer(IlsRecordsIndexer):
 
     record_cls = PatronTransaction
 
+    def index(self, record):
+        """Indexing a record."""
+        # Indexing of events created in the extension
+        for pid in record.event_pids:
+            PatronTransactionEvent.get_record_by_pid(pid).reindex()
```

Comment:
Hopefully `get_record_by_pid` will not response `None` otherwise we will get an exception here.